pkg/mgmt/lvmnode/builder.go

```diff
@@ -67,91 +75,29 @@ type NodeController struct {
 	ownerRef metav1.OwnerReference
 }
 
-// NodeControllerBuilder is the builder object for controller.
```

Comment:
Why are we removing the builder code?
The builder pattern may be more useful here as the Controller may be extended to different configurations easily, without breaking existing code (i.e. by altering function signature).

Reply:
I did check the previous implementation . I felt it to be more complex than it needs to be. In previous implementation as well incase we need more functionality we would need to add new functions and use them in builder call. Now with this where one function builds whole controller we might just have to add new args in function definition . 

This is the standard procedure i have seen across many controller implementation. Reduces multiple function calls as well.

Reply:
Long running list of function arguments affect readability.
Builder pattern makes the Controller build parameters more readable. This is a pattern we have followed throughout this repository for better readability, when building instances of types with many parameters.

**E.g.**
If one tries to extend the code to include another controller with different configuration, and a different use-case, this single function won't be usable for that. That person will have to add another function for his use-case.
With the builder-pattern, we can accomodate most use-cases.

We are not optimising for performance, we are not optimising for lines of code. We are optimising for readability.

Reply:
Let's remove the builder. 👍